src/style.css

```diff
@@ -0,0 +1,56 @@
+.big-search-form {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
```

Comment:
The better practice is to use HEX instead of RGBA, because it is a more compact recording

Reply:
you can convert it to hex anyway, you can do it directly in VS code
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96265844/89fac845-5364-4827-b381-54e2b5ff63dc)

Reply:
Thank you very much, I'll know.